client/src/app/app.component.spec.ts

```diff
@@ -1,12 +1,23 @@
 import { TestBed } from '@angular/core/testing';
 import { RouterTestingModule } from '@angular/router/testing';
 import { AppComponent } from './app.component';
+import { EmployeeAppModule } from './employee-app/employee-app.module';
+import { MatToolbarModule } from '@angular/material/toolbar';
+import { MatIconModule } from '@angular/material/icon';
+import { EmployeeAppRootComponent } from './employee-app/employee-app-root/employee-app-root.component';
 
 describe('AppComponent', () => {
-  beforeEach(() => TestBed.configureTestingModule({
-    imports: [RouterTestingModule],
-    declarations: [AppComponent]
-  }));
+  beforeEach(() =>
+    TestBed.configureTestingModule({
+      imports: [
+        RouterTestingModule,
+        EmployeeAppModule,
+        MatToolbarModule,
+        MatIconModule,
```

Comment:
Nice job figuring out about the updated imports! Was it hard to debug this? (It took me probably 30-60 minutes to figure out what was going on with this.)

Reply:
Actually FuMing told me that I also needed to update in the tests.